src/global/less/corporate-ui/core.less

```diff
@@ -42,7 +42,11 @@
 
 // Custom-classes for corporate-ui
 @import 'custom-classes/index';
-@import 'custom-classes/cards';
+
+// Bootstrap 4 slow upgrade
```

Comment:
Might be better to create a new bs4 folder where we put these.
They are not custom classes.